src/tabs/tab.jsx

```diff
@@ -97,12 +103,13 @@ const Tab = React.createClass({
       onTouchTap,
       selected,
       style,
+      tabIndex,
       value,
       width,
       ...other,
     } = this.props;
 
-    const styles = this.mergeStyles({
+    const styles = this.prepareStyles({
```

Comment:
`mergeStyles` was better. `prepareStyles` should only be called before providing it to the `style` property.